### ISSUE TITLE
Use custom APIs on Buidlfy using iExec Oracle Factory

### ISSUE THREAD

**Issue by aayushmahapatra** [buidlfyHq/buidlfy-improvements#1]
## TL;DR
Use custom APIs on Buidlfy using iExec Oracle Factory.

## About iExec Oracle Factory
The iEexc Oracle Factory is a developer interface that allows you to build an oracle on Blockchain from scratch, in just a few minutes. You can use either preset APIs or your custom APIs and create these oracles directly from your browser without any specialised coding knowledge.
> Here's the link to their website: https://oracle-factory.iex.ec/gallery

## Proposal
- Let's say you want to show the temperature of a place in your Dapp from your smart contract.
- In the current scenario, there's no way to do that other than hard-coding the value.
- But with iExec Oracle factory, you can create an oracle from a weather api and use that in your Dapp.
- Thus, I propose to integrate **iExec by Oracle Factory** whereby we will be able to use these oracles in Buidlfy. 
- This will allow us to use any API available on the web in our Dapps and will unlock a whole new range of blockchain use cases.

## Implementation
- One possible way of doing that is by providing a similar functionality that we use while using smart contracts.
- We can add the Oracle ID and Select an element on the page where we want to show the result.
- This oracle can be triggered either with a button or on page load.

## Conclusion
With this integration, we will be able to use the terabytes of valuable data existing on the traditional ‘Web 2.0’ internet in our Dapps. created using Buidlfy, in just a few minutes, without having to write a single piece of code.
